mods/ra/maps/allies-09a/allies09a.lua

```diff
@@ -0,0 +1,119 @@
+--[[
+   Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+
+lstReinforcements =
+{
+	actors = { "mcv"},
+	entryPath = { AlliedMCVEntry.Location, Unload1.Location },
+	exitPath = { AlliedMCVEntry.Location }
+}
+
+ExtractionHelicopterType = "tran"
+ExtractionPath = { HeliWP01.Location, HeliWP02.Location, HeliWP03.Location }
+
+kosyginType = "gnrl"
+contectKosygin = false
```

Comment:
Style nit: Global variables start with an upper case letter: `KosyginType` and `KosyginContacted` (also contact instead of contect.)

Reply:
Fixed at [c9bbea5](https://github.com/OpenRA/OpenRA/commit/c9bbea5b361b63990c5c073f6ac271692cc2ea62)